src/endpoints.go

```diff
@@ -12,16 +13,30 @@ import (
 const CONTAINER_STOP_TIMEOUT = 10 // seconds
 
 var (
-	daemonPort   int    = 2375
-	daemonHost   string = fmt.Sprintf("http://127.0.0.1:%d", daemonPort)
-	dockerClient *docker.Client
+	daemonPort    int = 2375
+	daemonTLSPort int = 2376
+	dockerClient  *docker.Client
 )
 
 func init() {
-	daemonHost = fmt.Sprintf("http://%s:%d", getDaemonHostIp(), daemonPort)
+	var daemonHost string
+	var client *docker.Client
+	var err error
+	certPath := os.Getenv("DOCKER_CERT_PATH")
+	if certPath == "" {
+		daemonHost = fmt.Sprintf("tcp://%s:%d", getDaemonHostIp(), daemonPort)
+		client, err = docker.NewClient(daemonHost)
+	} else {
+		daemonHost = fmt.Sprintf("tcp://%s:%d", getDaemonHostIp(), daemonTLSPort)
+		log.Info("Docker cert path: %s", certPath)
+		ca := fmt.Sprintf("%s/ca.pem", certPath)
```

Comment:
thoughts on moving these to just envars instead? Like:

- `TLS_CA_CERT`
- `TLS_CERT`
- `TLS_KEY`

Same names as the daemon and client expects, then we no longer need `DOCKER_CERT_PATH` and they can place it wherever they like and no file name hardcoding.